### PR TITLE
docs: add some useful tips for testing

### PR DESCRIPTION
**Pull request by ematipico**
## Changes

This PR adds some useful tips to our contribution guide regarding testing.

## Testing

N/A

## Docs

N/A

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -111,6 +111,52 @@ pnpm run test -m "$STRING_MATCH"
...
 ```
 
+#### Running a single test
+
+Sometimes you want to run a single test case (`it` or `describe`) or a single test file. You can do so by using Node.js. 
+
+To run a single test file, for example `test/astro-basic.test.js`:
+
+```shell
+node --test test/astro-basic.test.js
```
Comment (florian-lefebvre) [inline CONTRIBUTING.md:121]: I've always been running without --test, I'm curious to know what difference it makes

**Review by florian-lefebvre — APPROVED** [review]
Really great addition!

File: CONTRIBUTING.md (hunk 1 above)
Comment (ematipico) [reply CONTRIBUTING.md:121]: I don't think there's a difference. However, it's possible that the flag does some optimisations underhood when combined with other options. 

Also, considering that's the [recommended way](https://nodejs.org/api/test.html#running-tests-from-the-command-line), we should suggest it too.

**Comment by ascorbic** [comment]
Nice!

**Review by bluwy — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -111,6 +111,52 @@ pnpm run test -m "$STRING_MATCH"
...
+```
+
+If you wish to run a single test case, you have to postfix `it` and `describe` functions with `.only`:
```
Comment (matthewp) [inline CONTRIBUTING.md:124]: i still struggle with this so much

**Review by matthewp — APPROVED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (ematipico) [reply CONTRIBUTING.md:124]: Me too, it's not a great DX

**Pull request merged**